src/main/markdown/lifeofanissue.md

```diff
@@ -1,7 +1,7 @@
 Life of an issue
 ===
 
-The GWT project maintains a [public issue tracker](http://code.google.com/p/google-web-toolkit/issues) where you can report bugs and request features for GWT.
+The GWT project maintains a [public issue tracker](https://github.com/gwtproject/gwt/issues/) where you can report bugs and request features for GWT.
```

Comment:
Nit: GitHub doesn't put the trailing slash in their own links (though it also works); and you didn't put it in the other link below. I think we should at least be consistent.